portalnet/src/overlay_service.rs

```diff
@@ -2171,7 +2179,10 @@ where
         &mut self,
         target: TContentKey,
         callback: Option<oneshot::Sender<FindContentResult>>,
+        is_trace: bool,
     ) -> Option<QueryId> {
+        info!("Starting query for content key: {}", target);
```

Comment:
We might want to reduce this to debug or drop it altogether, if we notice it being too little noisy, but I'm pretty happy to let that evolve over time, and be just a little generous with logs, especially with new features.